src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,1398 @@
     type: command
     short-summary: Creates a remote connection using a tcp tunnel to your web app
 """
+
+helps['functionapp hybrid-connection'] = """
```

Comment:
this help is just for the group functionapp hybrid-connection - so keep the description focused to just that. & move the list examples & description to the list help